lib/engine/game/g_18_scan/game.rb

```diff
@@ -228,26 +230,52 @@ class Game < Game::Base
               { 'nodes' => ['city'], 'pay' => 4, 'visit' => 4, 'multiplier' => 2 },
               { 'nodes' => ['town'], 'pay' => 0, 'visit' => 99 },
             ],
+            available_on: '5E',
             price: 800,
             num: 2,
           },
         ].freeze
 
         def setup
-          # SJ cannot float until phase 5
-          sj.floatable = false
-
           # Minors come with a trade-in share of SJ
           minors.each do |minor|
             share = sj_share_by_minor(minor.name)
             share.buyable = false
             share.counts_for_limit = false
+
+            # Reserve token locations for minors
+            cities = hex_by_id(minor.coordinates).tile.cities
+
+            if minor.city
+              cities[minor.city].add_reservation!(minor)
+            else
+              cities.first.add_reservation!(minor)
+            end
           end
         end
 
+        def init_corporations(stock_market)
+          game_corporations.map do |corporation|
+            G18Scan::Corporation.new(
+              self,
+              min_price: stock_market.par_prices.map(&:price).min,
+              capitalization: self.class::CAPITALIZATION,
+              **corporation.merge(corporation_opts),
+            )
+          end
+        end
+
+        def corporation_opts
+          two_player? && @optional_rules&.include?(:two_player_share_limit) ? { max_ownership_percent: 70 } : {}
+        end
+
+        def all_corporations
+          minors + corporations
```

Comment:
this should probably be refactored out and made default

Reply:
Is this okay that all other games that haven't overridden this method start displaying minors on Spreadsheet tab? (eg 1846)

Reply:
These 6 use `minors + corporations`:
1873
1893
18_eu
18_mag
18_mex
18_scan

Other 10 games with minors use default method (only corporations)
1824
1828
1835
1846
18_los_angeles
18_mo
18_new_england
18_new_england_north
18_sj
2038

There are other games that treat minors as corporations manually (I know that 1861 implementation does this)

Reply:
ok, let's keep it as is then, ignore me thanks